<ui version="4.0" >
 <class>BarCharDisplayEditor</class>
 <widget class="QWidget" name="BarCharDisplayEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>203</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="ViewGroup" >
     <property name="title" >
      <string>View</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>6</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="ViewData" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Toggle visibility of this dataset's geometry.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string>Visible</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>X Axis</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="1" >
       <widget class="QComboBox" name="PointComponent" >
        <item>
         <property name="text" >
          <string>X</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Y</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Z</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="labelXArrayName" >
        <property name="text" >
         <string>Array Name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QComboBox" name="XArrayName" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="labelPointComponent" >
        <property name="text" >
         <string>Component</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2" >
       <widget class="QCheckBox" name="UsePoints" >
        <property name="toolTip" >
         <string>&lt;html>When checked the chart will use points instead of point data on the X axis.&lt;/html></string>
        </property>
        <property name="text" >
         <string>Use Points</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2" >
       <widget class="QPushButton" name="EditColorMapButton" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Edit Color Map</string>
        </property>
        <property name="visible" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Y Axis</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="1" >
       <widget class="QComboBox" name="YArrayName" />
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Array Name</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>UsePoints</sender>
   <signal>toggled(bool)</signal>
   <receiver>PointComponent</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>101</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel" >
     <x>143</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UsePoints</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelPointComponent</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>101</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel" >
     <x>59</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UsePoints</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelXArrayName</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>101</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel" >
     <x>59</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UsePoints</sender>
   <signal>toggled(bool)</signal>
   <receiver>XArrayName</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>101</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel" >
     <x>143</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
